inc/saitunnel.h

```diff
@@ -647,6 +676,35 @@ typedef enum _sai_tunnel_attr_t
      */
     SAI_TUNNEL_ATTR_TERM_TABLE_ENTRY_LIST,
 
+    /**
+     * @brief Tunnel EXP mode (pipe or uniform model)
+     *
+     * @type sai_tunnel_exp_mode_t
+     * @flags CREATE_ONLY
+     * @default SAI_TUNNEL_DSCP_MODE_UNIFORM_MODEL
+     */
+    SAI_TUNNEL_ATTR_ENCAP_EXP_MODE,
+
+    /**
+     * @brief Tunnel EXP value (3 bits)
+     *
+     * @type sai_uint8_t
+     * @flags MANDATORY_ON_CREATE | CREATE_ONLY
+     * @condition SAI_TUNNEL_ATTR_ENCAP_EXP_MODE == SAI_TUNNEL_EXP_MODE_PIPE_MODEL and SAI_TUNNEL_ATTR_TYPE == SAI_TUNNEL_TYPE_MPLS
+     */
+    SAI_TUNNEL_ATTR_ENCAP_EXP_VAL,
+
+    /**
+     * @brief Tunnel EXP mode (pipe or uniform model)
+     *
+     * Default SAI_TUNNEL_EXP_MODE_UNIFORM_MODEL
+     *
+     * @type sai_tunnel_exp_mode_t
+     * @flags CREATE_ONLY
+     * @default SAI_TUNNEL_DSCP_MODE_UNIFORM_MODEL
+     */
+    SAI_TUNNEL_ATTR_DECAP_EXP_MODE,
```

Comment:
Is there a need to specify encap and decap mode separately. Is it possible to have just 'SAI_TUNNEL_ATTR_EXP_MODE'

Reply:
There is SAI_TUNNEL_ATTR_DECAP_DSCP_MODE and equivalently added SAI_TUNNEL_ATTR_DECAP_EXP_MODE